cmake/modules/SearchRootCoreDeps.cmake

```diff
@@ -130,16 +221,20 @@ else()
 
   find_package(NumPy)
 
-  set(PYTHON_UNDER_VERSION_STRING "${PYTHON_VERSION_MAJOR}_${PYTHON_VERSION_MINOR}")
+  set(PYTHON_EXECUTABLE_Development_Main "${PYTHON_EXECUTABLE}")
```

Comment:
I see, in the case of old style build (old CMake), we still need to set these because PyROOT and PyMVA rely on them to build, right?